pkg/controller/kogitoinfra/kogitoinfra_controller.go

```diff
@@ -65,14 +65,20 @@ func add(mgr manager.Manager, r reconcile.Reconciler) error {
 		return err
 	}
 
-	watchOwnedObjects := []runtime.Object{&corev1.Secret{}, &v1.Infinispan{}}
+	watchOwnedObjects := []runtime.Object{&corev1.Secret{}, &infinispanv1.Infinispan{}}
 	ownerHandler := &handler.EnqueueRequestForOwner{
 		IsController: true,
 		OwnerType:    &appv1alpha1.KogitoInfra{},
 	}
 	for _, watchObject := range watchOwnedObjects {
 		err = c.Watch(&source.Kind{Type: watchObject}, ownerHandler)
 		if err != nil {
+			if kindErr, ok := err.(*meta.NoKindMatchError); ok {
+				if kindErr.GroupKind.Group == infinispanv1.SchemeGroupVersion.Group {
+					log.Warnf("Tried to watch Infinispan CRD, but failed. Infinispan deployment won't be available.")
```

Comment:
Are the warn logs supposed to appear somewhere ? like in operator logs ?

Reply:
Appears in the beginning of the operator log

Reply:
ok got it ;)